eduard.iskandaryan/repo/Lesson_3/exercise_03_05/exercise_03_05.py

```diff
@@ -0,0 +1,2 @@
+x = int(input("Enter a number: "))
+print("The", "square", "of", x, "is", x**2, sep=" ", end=".")
```

Comment:
LW: Use f-string formatting type to print the message. Also, use spaces before and after the arithmetic operator.